src/status_im/components/toolbar_new/view.cljs

```diff
@@ -13,10 +13,17 @@
             [status-im.components.context-menu :refer [context-menu]]
             [status-im.components.toolbar-new.actions :as act]
             [status-im.components.toolbar-new.styles :as st]
-            [status-im.accessibility-ids :as id]
             [status-im.utils.platform :refer [platform-specific]]
             [reagent.core :as r]))
 
+(defn create-nav-button [nav-action]
```

Comment:
we could desctruct `nav-action` in params vector, it would look less hairy imo, something like
```clojure
(defn create-nav-button
  [{:keys [handler accessibility-label] nav-image :image}]
  [touchable-highlight
   (merge {:style    st/toolbar-button
           :on-press handler}
          (when accessibility-label
            {:accessibility-label accessibility-label}))
   [view [image nav-image]]])
```
when this ns will be refactored we can get rid of `nav-image`